No versions avaliable for tesla-screenplay-playwright🐞 <title>

### What is the issue?

Al instalar la librería tesla segun el comando descrito en la documentación oficial me muestra un erro que no hay versiones disponibles

### Log output

PS C:\Users\CCXC Administrador\Documents\FOLDER 2024\FOLDER 2023\PORVENIR\Automatización\Auth Tesla> npm install --save-dev testla-screenplay-playwright
npm ERR! code ENOVERSIONS
npm ERR! No versions available for testla-screenplay-playwright

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\CCXC Administrador\AppData\Local\npm-cache\_logs\2024-03-19T17_38_29_861Z-debug-0.log

### Steps to reproduce

_No response_

### Testla version

18.16.0

### Browser

Chromium

--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ In order to use testla you need to [install](https://playwright.dev/docs/intro) 
 Get started by installing Testla Screenplay for Playwright using npm. 
 
 ```bash
-npm install --save-dev testla-screenplay-playwright
+npm install --save-dev @testla/screenplay-playwright
 ```
 
 ## 👨‍🏫 Learn Testla

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@testla/screenplay-playwright",
-  "version": "1.5.0",
+  "version": "1.5.1",
   "description": "Implementation of the Playwright abilities, actions and questions.",
   "main": "index.js",
   "types": "index.d.ts",

